addons/mrp/models/mrp_workorder.py

```diff
@@ -114,6 +123,26 @@ class MrpWorkorder(models.Model):
         'finished_workorder_id', string='By-products')
     allowed_lots_domain = fields.One2many(comodel_name='stock.production.lot', compute="_compute_allowed_lots_domain")
 
+    @api.depends('leave_id')
+    def _compute_dates_planned(self):
+        for wo in self:
+            wo.update({
```

Comment:
why update in a compute?

Reply:
![](https://media.giphy.com/media/u953mQmL1azte/giphy.gif)